runtime/runtime/src/lib.rs

```diff
@@ -1070,6 +1072,39 @@ impl Runtime {
         Ok(())
     }
 
+    pub fn apply_migrations(
+        &self,
+        state_update: &mut TrieUpdate,
+        migration_data: &Arc<MigrationData>,
+        protocol_version: ProtocolVersion,
+    ) -> Result<Gas, StorageError> {
+        #[cfg(feature = "protocol_feature_fix_storage_usage")]
+        let mut gas_used: Gas = 0;
+        #[cfg(not(feature = "protocol_feature_fix_storage_usage"))]
+        let gas_used: Gas = 0;
+        #[cfg(feature = "protocol_feature_fix_storage_usage")]
+        if ProtocolFeature::FixStorageUsage.protocol_version() == protocol_version {
+            for (account_id, delta) in &migration_data.storage_usage_delta {
+                match get_account(state_update, account_id)? {
+                    Some(mut account) => {
+                        // Storage usage is saved in state, hence it is nowhere close to max value
+                        // of u64, and maximal delta is 4196, se we can add here without checking
+                        // for overflow
```

Comment:
Maybe checked add would be just easier than long comment why it's not needed :)?